CurrencyConverterService/Service.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using CurrencyConverterService.Models;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace CurrencyConverterService
+{
+    class Service
+    {
+        public static bool serviceWork = true;
+        public const int interwal = 100000;
```

Comment:
static members and constants should start from upper letter. 
public members also.
private should be started from '_' symbol